pages/Favorites/components/FavoritesList/components/Item/components/CTAButtons/connector.js

```diff
@@ -11,6 +11,7 @@ import { MODAL_VARIANT_SELECT } from '@shopgate/pwa-ui-shared/Dialog/constants';
  */
 const mapStateToProps = state => ({
   isBaseProduct: productId => isBaseProduct(state, productId),
+  isOrderable: productId => isOrderable(state, productId),
```

Comment:
If you have the productId already in the props you can pass it into the selectors from the props instead of creating a callable function. That is redundant and then the selectors don't make sense:
```
isOrderable: isOrderable(state, props.productId)
```
Same applies for the `isBaseProduct` prop.

Reply:
True. Done.